openedx/core/djangoapps/lang_pref/api.py

```diff
@@ -14,6 +16,20 @@
 Language = namedtuple('Language', 'code name')
 
 
+def header_language_selector_is_enabled():
+    """Return true if the header language selector has been enabled via settings or site-specific configuration."""
+    setting = get_value('SHOW_HEADER_LANGUAGE_SELECTOR', settings.FEATURES.get('SHOW_HEADER_LANGUAGE_SELECTOR', False))
+
+    # The SHOW_LANGUAGE_SELECTOR setting is deprecated, but might still be in use on some installations.
+    deprecated_setting = get_value('SHOW_LANGUAGE_SELECTOR', settings.FEATURES.get('SHOW_LANGUAGE_SELECTOR', False))
+    return setting or deprecated_setting
```

Comment:
Here, I believe it makes sense to return the new setting if it has been set to True, and otherwise fall back to the deprecated setting. That way, if the deprecated setting is explicitly set to True somewhere we will honor it. If anyone explicitly uses the new setting to configure the language selector to something other than the default (false), they should take care to remove all references to the old setting. For example, if they update lms.env.py and replace  `'SHOW_LANGUAGE_SELECTOR':True` with `'SHOW_HEADER_LANGUAGE_SELECTOR':True`, they should also take care to update any site-specific configuration referencing `'SHOW_LANGUAGE_SELECTOR'` @andy-armstrong , @jibsheet does this seem reasonable to you?